src/test/java/se/sundsvall/contract/api/model/LandLeaseContractTest.java

```diff
@@ -106,7 +106,7 @@ void testBuilderMethods() {
 		final var leaseholdType = Leasehold.builder().build();
```

Comment:
Details, but should maybe be renamed to `leasehold`?

Reply:
Definitely